### DISCUSSION TITLE
Support Azure Table as a new storage plugin

### DISCUSSION THREAD

**Discussion by luxianlong** [JanusGraph/janusgraph#4380] [Ideas]
Hi, all,
       I'm an Engineer from Microsoft. We've developed an storage plugin based on azure table. And we're willing to contribute it back to the community. I'm wondering if there is any blocker for such feature which supports JanusGraph on a public cloud service? Thanks.

**Comment by porunov** [comment]
Hi! There is no any blocker, but usually it is preferred to release a storage adapter as a separate repository instead of the main codebase. If there are developers who are willing to support this storage adapter in JanusGraph then it should be good to be moved under the JanusGraph account. Otherwise, it should be suitable to be under the organisation who contributes this storage adapter. 
Here is an Aerospike storage adapter for reference which is managed solely by Playtika: 
https://github.com/PlaytikaOSS/aerospike-janusgraph-storage-backend

**Reply by luxianlong** [reply]
Thanks for sharing insights @porunov  . I will try to put it under organization Microsoft code repository.
